_posts/research/subteams/2021-03-16-algorithmic-social-hri.md

```diff
@@ -58,19 +58,15 @@ Large language models (LLMs) like ChatGPT, in contrast, are extraordinarily flex
 Our ongoing work seeks to narrow the gap between human ingenuity and algorithmic efficiency by enabling users to shape, update, and modify TAS systems entirely via LLM-mediated, interactive dialogue.
 We are confident that this approach will not only enhance the efficiency and flexibility of task management, but also democratizes the use of advanced TAS algorithms.
 
-## 1.3 Reference Dependent Risk Attitudes in HRI
+## 1.3 Predictability in HRI
```

Comment:
Move this all up to section 1.2. We list them in order of recency and since your THRI paper is about to be published, it should be up.

Reply:
(and move the current setion 1.2 down :-) )